CMakeLists.txt

```diff
@@ -180,7 +175,15 @@ if(UNIX AND NOT APPLE)
   mark_as_advanced(F3D_INSTALL_THUMBNAILER_FILES)
 endif()
 
-include(cmake/installing.cmake)
+# Installing licenses
+set(F3D_LIC_DIR ".")
+
+if (UNIX AND NOT APPLE AND NOT ANDROID)
+  set(F3D_LIC_DIR ${CMAKE_INSTALL_DOCDIR})
+endif()
+
+install(FILES LICENSE.md doc/THIRD_PARTY_LICENSES.md
```

Comment:
It would be better to put licenses files where they are used (e.g. assimp in assimp plugin) so the licenses are installed only if it is used (ie when assimp plugin is enabled)

Reply:
assimp license is not in THIRD_PARTY_LICENSES.md anymore.
This project do not distribute assimp, the superbuild-does and assimp license is distributed in the package.

THIRD_PARTY_LICENSES.md contains only jsoncpp and cxxopts (check the diff), which could arguably be put in the application folder as only application rely on them. Do you think this is worth it ?